src/Routes.js

```diff
@@ -78,25 +78,23 @@ const Routes = (props) => (
     <AuthenticatedRoute path="/summary_votes" component={SummaryVotes} props={props} rolesAllowed={[USER_ROLES.all]} />
     <AuthenticatedRoute path="/researcher_console" component={ResearcherConsole} props={props} rolesAllowed={[USER_ROLES.researcher]} />
     {
-      props.env === 'dev' ? <AuthenticatedRoute path="/new_researcher_console" component={NewResearcherConsole} props={props} rolesAllowed={[USER_ROLES.researcher]}/>
-        : ''
+      props.env !== 'prod' ? <AuthenticatedRoute path="/new_researcher_console" component={NewResearcherConsole} props={props} rolesAllowed={[USER_ROLES.researcher]}/>
+        : <AuthenticatedRoute path="/researcher_console" component={ResearcherConsole} props={props} rolesAllowed={[USER_ROLES.researcher]} />
     }
```

Comment:
This has the effect of `researcher_console` always pointing to `ResearcherConsole` because line 79 is always true. What I think we need to do is in the case of not prod, point both `researcher_console` and `new_researcher_console` to `NewResearcherConsole`. Only in prod should we see `path="/researcher_console" component={ResearcherConsole}` in effect.

Reply:
Ah, forgot to delete line 79. I

Reply:
I think line 82 should be an empty string as the last part of the ternary.